### ISSUE TITLE
Confirmation when adding Token in TokenSelector

### ISSUE THREAD

**Issue by Velenir** [gnosis/dex-react#854]
> that is best to ask for confirmation to the user OR at least show somewhere a warning message (informing of the dangers of adding a token) and a link to etherscan.
  -- @anxolin 

Add a confirmation modal similar or even reusing one from _Add token from URL_

**Comment by Velenir** [comment]
Tried adding i/t again and ran into edge case upon edge case
Major problem being react-select resetting input text on every internal action 
```ts
type InputActionTypes =
  | 'set-value'
  | 'input-change'
  | 'input-blur'
  | 'menu-close';
```

and we hack around with custom input which results in `menu-close` firing on input focus

**Comment by Velenir** [comment]
POC in #882

**Comment by Velenir** [comment]
Closed by #866
